Runtime/Game/Requests/AssetRequest.cs

```diff
@@ -187,6 +194,17 @@ public class LootLockerActivateRentalAssetResponse : LootLockerResponse
     {
         public int time_left { get; set; }
     }
+
+    public class LootLockerGrantAssetResponse : LootLockerResponse
+    {
+        public int id { get; set; }
+        public int asset_id { get; set; }
+        public int asset_variation_id { get; set; }
+        public int asset_rental_option_id { get; set; }
+        public string asset_ulid { get; set; }
```

Comment:
Just want to double-check, is it ulid or uuid?
This https://ref.lootlocker.com/game-api/#asset
shows uuid and not ulid, but this response might be different.

Reply:
It is indeed asset_ulid :)